### PR TITLE
Update windows-autopilot-hybrid.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document as the navigation has changed

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2253

**Comment by PRMerger20** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 1]
```diff
@@ -246,9 +246,10 @@ It takes about 15 minutes for the device profile status to change from *Not assi
     - **Name**: Enter a descriptive name for the new profile.
     - **Description**: Enter a description for the profile.
     - **Platform**: Select **Windows 10 and later**.
-    - **Profile type**: Select **Domain Join**.
-3. Select **Settings**, and then provide a **Computer name prefix**, **Domain name**.
-4. (Optional) Provide an **Organizational unit** (OU) in [DN format](/windows/desktop/ad/object-names-and-identities#distinguished-name). Your options include:
+    - **Profile type**: Select **Tempplates**, Choose the Template name **Domain Join** and click on **Create**
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:249]:
```suggestion
    - **Profile type**: Select **Templates**, choose the template name **Domain Join** and select **Create**.
```

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 2]
```diff
@@ -246,9 +246,10 @@ It takes about 15 minutes for the device profile status to change from *Not assi
...
+3. Enter the **Name**, **Description** and Click on **Next**
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:250]:
```suggestion
3. Enter the **Name** and **Description** and select **Next**. 
```

File: memdocs/autopilot/windows-autopilot-hybrid.md [hunk 3]
```diff
@@ -246,9 +246,10 @@ It takes about 15 minutes for the device profile status to change from *Not assi
     - **Name**: Enter a descriptive name for the new profile.
     - **Description**: Enter a description for the profile.
     - **Platform**: Select **Windows 10 and later**.
-    - **Profile type**: Select **Domain Join**.
-3. Select **Settings**, and then provide a **Computer name prefix**, **Domain name**.
-4. (Optional) Provide an **Organizational unit** (OU) in [DN format](/windows/desktop/ad/object-names-and-identities#distinguished-name). Your options include:
+    - **Profile type**: Select **Tempplates**, Choose the Template name **Domain Join** and click on **Create**
+3. Enter the **Name**, **Description** and Click on **Next** 
+5. Provide a **Computer name prefix**, **Domain name**.
```
Comment (JohanFreelancer9) [inline memdocs/autopilot/windows-autopilot-hybrid.md:251]:
```suggestion
5. Provide a **Computer name prefix** and **Domain name**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger14** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dougeby Kindly help. Thanks

**Pull request merged**